debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/SourceInfoTest.java

```diff
@@ -34,4 +35,10 @@ public void versionIsPresent() {
     public void connectorIsPresent() {
         assertThat(source.source().getString(SourceInfo.DEBEZIUM_CONNECTOR_KEY)).isEqualTo(Module.name());
     }
+
+    @Test(expected = Test.None.class /* no exception expected */)
```

Comment:
Why not just omit the `expected` then if no exception is expected?

Reply:
I didn't know that existed

Reply:
I'm used to scalatest where you write out that you except no exception

Reply:
Ah, I see. Never used Scalatest before. Always learning something new here :)